samples/SessionSample/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
We're very inconsistent about it. Some samples require the customizations, so I include them.